WcaOnRails/app/views/static_pages/disclaimer.html.erb

```diff
@@ -0,0 +1,12 @@
+<% provide(:title, t('disclaimer.title')) %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <ul>
+    <li>The World Cube Association (WCA) works to verify that appropriate WCA Regulations are followed for the competition portion of WCA-recognized competitions.</li>
```

Comment:
Shouldn't we localize this?

Reply:
I'm not sure. Perhaps we should ask the Board. There a couple pages non-localizable (privacy one, I think).

Reply:
The Board said this should be English-only.
I'll remove the title key as well.